content/source/docs/cloud/workspaces/agent.html.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: "cloud"
+page_title: "Terraform Cloud Agents - Workspaces - Terraform Cloud and Terraform Enterprise"
+---
+
+# Terraform Cloud Agents
+
+-> **Note:** Terraform Cloud Agents are a paid feature, available as part of the **Terraform Cloud for Business** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
+
+Terraform Cloud Agents are a solution to allow Terraform Cloud to communicate with isolated, private or on-premises infrastructure. By deploying the lightweight agent within a specific network segment, you can establish a simple connection between your environment and Terraform Cloud which allows for provisioning operations and management. This is useful for on-premises infrastructure types such as vSphere, Nutanix, OpenStack, enterprise Networking providers, and anything you might have in a protected enclave.
```

Comment:
change to `enterprise networking` (lowercase the N, it's awkward and not aligned with the style guide)

Reply:
This one is already caught above